pkg/announce/github_page.go

```diff
@@ -104,11 +105,19 @@ type GitHubPageOptions struct {
 	Substitutions map[string]string
 }
 
+type SBOMFormat string
+
+const (
+	FormatJSON     = "json"
```

Comment:
```suggestion
	FormatJSON     SBOMFormat = "json"
```
Or avoid the type `SBOMFormat` at all and stick to strings.

Reply:
Ugh good catch, I kept it because I think we may have to use other string values as we wire the format deeper.

Thanks!